osdk/tools/docker/gen_dockerfile.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: MPL-2.0
+
+# The Dockerfile generator for OSDK.
+
+output_dir="ubuntu"
+base_image="ubuntu:22.04"
+install_qemu_ovmf=true
```

Comment:
I don't think `true` and `false` are special values in Bash scripts. You are writing as if they are anything more than a string that happens to be "true" or "false".